clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/ops/servergroup/ResizeDcosServerGroupAtomicOperation.groovy

```diff
@@ -44,12 +43,14 @@ class ResizeDcosServerGroupAtomicOperation implements AtomicOperation<Void> {
 
     task.updateStatus BASE_PHASE, "Setting size to $size..."
 
-    def app = new App()
+    def app = maybeApp.get()
```

Comment:
We'll need to make sure to revert this once we're using the proper marathon-client version that fixes the problem this works around (or wait to merge this until afterwards).